src/dash-table/components/Cell/index.tsx

```diff
@@ -20,15 +20,15 @@ export default class Cell extends Component<ICellProps> {
     }
 
     render() {
-        const { classes, property, style } = this.propsWithDefaults;
+        const { attributes, classes, style } = this.propsWithDefaults;
 
         return (<td
             ref='td'
             children={(this as any).props.children}
             tabIndex={-1}
             className={classes}
             style={style}
-            data-dash-column={property}
+            {...attributes}
```

Comment:
Pass all data-attributes structure for passing column id and row index information to the cell -- used by the tooltip decorator to figure out what it's hovering over